### ISSUE TITLE
[Bug]: Text selection colors overwrite system

### ISSUE THREAD

**Issue by cosorgen** [microsoft/fluentui#35394]
### Component

TextInput

### Package version

3.0.0-beta.130

### @microsoft/fast-element version

2.8.1

### Environment

```shell
System:
    OS: Windows 11 10.0.26200
    CPU: (32) x64 AMD Ryzen 9 5950X 16-Core Processor
    Memory: 41.79 GB / 63.91 GB
  Browsers:
    Chrome: 141.0.7390.108
    Edge: Chromium (141.0.3537.85)
    Firefox: 144.0 - C:\Program Files\Mozilla Firefox\firefox.exe
    Internet Explorer: 11.0.26100.1882
```

### Current Behavior

When you input text into the text field, then hit Cmd+A (select all). The color of the highlighted text is black with white text... These are linked to `colorNeutralForegroundInverted` and `colorNeutralBackgroundInverted`.

### Expected Behavior

Customers in Edge expect the highlight color of text selections to be the same across the system (usually blue by default). There is no way to implement this with the provided tokens since they are used in the design system elsewhere and cannot be changed to blue. This is also would be very cumbersome to dynamically account for users changing their highlight color at the system level.

### Reproduction

https://stackblitz.com/edit/vitejs-vite-5iyhaxw4?file=index.html

### Steps to reproduce

1. Type "hello" into the fluent text input.
2. Type Cmd/Ctrl + A to select the text. 3. Type
3. Observe text selection color
4. Type "hello" into the standard text input.
5. Type Cmd/Ctrl + A to select the text.
6. Observe text selection color.

### Are you reporting an Accessibility issue?

None

### Suggested severity

Urgent - No workaround and Products/sites are affected

### Products/sites affected

Edge

### Are you willing to submit a PR to fix?

yes

### Validations

- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [x] The provided reproduction is a minimal reproducible example of the bug.
